exporter/awsemfexporter/getJsonString

```diff
@@ -0,0 +1,60 @@
+import{
+	"encoding/json"
+	"fmt"
+}
+
+type kubernetesObj struct{
+	ContainerName string `json: "container_name"`
+	Docker internalDockerObj `json: "docker"`
+	Host string `json: "host"`
+	Labels internalLabelsObj `json: "labels"`
+	NamespaceName string `json: "namespace_name"`
+	PodId string `json: "pod_id"`
+	PodName string `json: "pod_name"`
+	PodOwners internalPodOwnersObj `json:"pod_owners"`
+	ServiceName string `json: "service_name"`
+}
+
+type internalDockerObj struct{
+	ContainerId string `json: "container_id"`
+}
+
+type internalLabelsObj struct{
+	App string `json: "app"`
+	PodTemplateHash string `json: "pod-template-hash"`
+}
+
+type internalPodOwnersObj struct{
+	OwnerKind string `json: "owner_kind"`
+	OwnerName string `json: "owner_name"`
+}
+
+
+func main(){
+	//create schema
+	schema := kubernetesObj{}
+	schema.ContainerName = "container_name"
+	schema.Docker =
+	internalDockerObj{
+		ContainerId: "container_id",
+	}
```

Comment:
is this a test file?

Reply:
This was a mistake, I have since removed it